ui/src/variables/reducers/index.ts

```diff
@@ -78,6 +78,13 @@ export const variablesReducer = (
         return
       }
 
+      case 'SELECT_VALUE': {
+        const {contextID, id, value} = action.payload
+        draftState.values[contextID].values[id].selectedValue = value
```

Comment:
This is a pretty long chain, is there any point where one of these 6 items could be undefined?

Reply:
good point